src/Bicep.Core.Samples/Files/InvalidExpressions_LF/main.diagnostics.bicep

```diff
@@ -457,13 +457,14 @@ var partialObject = {
 
 // dangling decorators - to make sure the tests work, please do not add contents after this line
 @concat()
+//@[1:7) [BCP147 (Error)] Function "concat" cannot be used as a decorator. |concat|
 @sys.secure()
 xxxxx
 //@[0:5) [BCP007 (Error)] This declaration type is not recognized. Specify a parameter, variable, resource, or output declaration. |xxxxx|
 
 
 @minLength()
```

Comment:
>minLength [](start = 1, length = 9)

Shouldn't this also include the original error?

Reply:
Yeah we should. I updated type checking so it doesn't short-circuit.